lib/events/dynamoevents/dynamoevents.go

```diff
@@ -208,7 +208,9 @@ func (cfg *Config) CheckAndSetDefaults() error {
 		cfg.UIDGenerator = utils.NewRealUID()
 	}
 
-	cfg.Endpoint = endpoint.CreateURI(cfg.Endpoint, cfg.Insecure)
+	if cfg.Endpoint != "" {
```

Comment:
Are these checks necessary? Looks like we check for non-empty inside `CreateURI`